composer/datasets/c4.py

```diff
@@ -140,7 +145,7 @@ class C4Dataset(IterableDataset):
         shuffle_buffer_size (int): If `shuffle=True`, samples are read into a buffer of this size (per-device), and randomly sampled from there to produce shuffled samples. (Default: `10000`)
```

Comment:
The line above seems too long. Try running `make style` to see if this gets fixed.